score-lang-compiler/src/test/java/org/openscore/lang/compiler/modeller/transformers/ForTransformerTest.java

```diff
@@ -112,15 +112,6 @@ public void testValidMapStatementWithoutSpaceAfterComma() throws Exception {
     }
 
     @Test
-    public void testValidMapStatementWithExpression() throws Exception {
```

Comment:
since we don't have `.items()` anymore, the tests simplifies to `k, v in dictionary` which is tested in  the testValidMapStatement() method